registry.js

```diff
@@ -180,14 +181,21 @@ registry.prototype.updateRegistry = function() {
 
     // if the registry does exist, update it
     ui.log('info', 'Updating registry cache...');
-    return asp(exec)('git fetch --all && git reset --hard origin/master', execOptions)
+    return asp(exec)('git fetch --all', execOptions)
     .then(function(stdout, stderr) {
       if (stderr)
         throw stderr;
+
+        //Doing this as a second command instead of combined IE.(git fetch --all && git reset --hard origin/master) because
+        //it seems like the execOptions dont get passed to the git reset command if executed as part of a combination
+        return asp(exec)( 'git reset --hard origin/master', execOptions)
```

Comment:
Are you saying this was needed because `strict-ssl` is actually supported by child_process?